funnel/assets/js/project_comments.js

```diff
@@ -0,0 +1,8 @@
+import { Utils, Comments } from './util';
+
+$(() => {
+  window.HasGeek.ProjectCommentsInit = function (pageUrl, sectionId) {
+    Utils.animateScrollTo($(sectionId).offset().top);
```

Comment:
Is scrolling on page load required?

Reply:
Removed since backend is sending with comment id as hash in the redirect url